apps/api/src/app/events/usecases/create-notification-jobs/create-notification-jobs.usecase.ts

```diff
@@ -142,4 +133,16 @@ export class CreateNotificationJobs {
 
     return steps;
   }
+
+  private calculateExpireAt(command: CreateNotificationJobsCommand) {
+    const delayedSteps = command.template.steps.filter(
+      (step) => step.template?.type === StepTypeEnum.DIGEST || step.template?.type === StepTypeEnum.DELAY
+    );
+
+    const delay = delayedSteps
+      .map((step) => this.delayService.calculateDelay(step, command.payload, command.overrides))
+      .reduce((sum, delayAmount) => sum + delayAmount, 0);
```

Comment:
Is our business logic to make the expireAt the exact sum of all the delay/digest steps? we don't want to add any error rate time? (like network latency etc) Or TTL Variant as we have on the caching? 
Or do we have the TTL expiration logic somewhere else in the code that takes it to the account?